docs/src/reference/asciidoc/appendix/release-notes/8.0.0.adoc

```diff
@@ -0,0 +1,62 @@
+[[eshadoop-8.0.0]]
+== Elasticsearch for Apache Hadoop version 8.0.0
+
+The following list are changes in 8.0.0 as compared to 7.17.0, and combines
+release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+[[breaking-8.0.0]]
+[float]
+=== Breaking Changes
+
+- Deprecate support for Spark 1.x, Scala 2.10 on Spark 2.x, Hadoop 1, Pig, and Storm
+https://github.com/elastic/elasticsearch-hadoop/pull/1845[#1845]
+
+Core::
+- Boost default scroll size to 1000
+https://github.com/elastic/elasticsearch-hadoop/pull/1429[#1429]
+
+[[new-8.0.0]]
+[float]
+=== Enhancements
+
+Build::
+- Upgrade build to use JDK 12 and current build tools
+https://github.com/elastic/elasticsearch-hadoop/pull/1351[#1351]
+
+- Build Hadoop with Java 17
+https://github.com/elastic/elasticsearch-hadoop/pull/1808[#1808]
+
+Build/Gradle::
+- Upgrade to and fix compatibility with Gradle 5.5
+https://github.com/elastic/elasticsearch-hadoop/pull/1333[#1333]
+
+- Update Gradle wrapper to 7.0
+https://github.com/elastic/elasticsearch-hadoop/pull/1641[#1641]
+
+- Setting Java boot classpath so that Scala 2.10 works on Gradle 7.2
+https://github.com/elastic/elasticsearch-hadoop/pull/1800[#1800]
+
+- Update Gradle wrapper to 7.3
+https://github.com/elastic/elasticsearch-hadoop/pull/1809[#1809]
+
+- Get rid of direct cross-project dependencies to avoid gradle warnings
+https://github.com/elastic/elasticsearch-hadoop/pull/1868[#1868]
+
+Core::
+- Setting X-Opaque-ID header for all reads and writes for mapreduce and spark
+https://github.com/elastic/elasticsearch-hadoop/pull/1770[#1770]
```

Comment:
This was actually first introduced in 7.16.0

Reply:
Thanks. Removed in https://github.com/elastic/elasticsearch-hadoop/pull/1902/commits/eaaae0aa85b8b6a4916773c152f3eb4d5cb774c4.